privacyidea/api/auth.py

```diff
@@ -309,6 +309,8 @@ def get_auth_token():
                                                     superuser_realms=
                                                     superuser_realms)
         details = details or {}
+        serials = ",".join([challenge_info["serial"] for challenge_info in details["multi_challenge"]]) \
```

Comment:
If You use `', '.join()` You'll have a space after the comma, looks better.

Reply:
If a space is added after the "," the link to the token contains a space that no longer works.